challenges/pt-br/new-backend-challenge/discount.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package discount;
+
+// Service that return mocked discount percentage.
+service Discount {
+  rpc GetDiscount(GetDiscountRequest) returns (GetDiscountResponse) {}
+}
+
+// productID used to represent a product. Ilustrative only.
+message GetDiscountRequest {
+  int32 productID = 1;
+}
+
+// The discount percentage is a fixed value.
+message GetDiscountResponse {
+  float percentage = 1;
+}
```

Comment:
Acho que aqui seria legal ter uma quebra de linha no final do arquivo, certo? "No newline at end of file"